### PR TITLE
[Tests-Only] used skeleton files more wisely on api share operations to features

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR:
- skeleton file/folders are used more wisely on the following features suite
  - apiShareOperationsToRoot
  - apiShareOperationsToShares

## Related Issue

- Part of https://github.com/owncloud/QA/issues/658

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiShareOperationsToRoot/accessToShare.feature [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+    Given these users have been created with default attributes and without skeleton files:
       | username |
       | Alice    |
       | Brian    |
 
   @smokeTest
   Scenario Outline: Sharee can see the share
     Given using OCS API version "<ocs_api_version>"
-    And user "Alice" has shared file "textfile0.txt" with user "Brian"
+    And user "Alice" has shared file "welcome.txt" with user "Brian"
```
Comment (individual-it) [inline tests/acceptance/features/apiShareOperationsToRoot/accessToShare.feature:13]: don't rely on having a `welcome.txt` that will not be the case for ocis or other special implementations like BayernBox

**Review by individual-it — CHANGES_REQUESTED** [review]
we need to do the opposite: wherever we have used `welcome.txt` we need to get rid of it
if the test needs only a single file, upload it in a Given step

File: tests/acceptance/features/apiShareOperationsToRoot/changingFilesShare.feature [hunk 2]
```diff
@@ -41,7 +41,7 @@ Feature: sharing
     Given using <dav-path-version> DAV path
     And user "Alice" has created folder "/shared"
     And user "Alice" has created folder "/shared/sub"
-    And user "Alice" has moved file "/textfile0.txt" to "/shared/sub/shared_file.txt"
+    And user "Alice" has moved file "/welcome.txt" to "/shared/sub/shared_file.txt"
```
Comment (Talank) [inline tests/acceptance/features/apiShareOperationsToRoot/changingFilesShare.feature:44]: welcome.txt is being used in several places.

**Comment by kiranparajuli589** [comment]
@individual-it @Talank addressed your comments. please review again.

File: tests/acceptance/features/apiShareOperationsToRoot/accessToShare.feature [hunk 3]
```diff
@@ -53,7 +59,7 @@ Feature: sharing
     Given using OCS API version "<ocs_api_version>"
     And group "grp1" has been created
     And user "Brian" has been added to group "grp1"
-    And user "Alice" has shared file "textfile0.txt" with group "grp1"
+    And user "Alice" has shared file "welcome.txt" with group "grp1"
```
Comment (Talank) [inline tests/acceptance/features/apiShareOperationsToRoot/accessToShare.feature:62]: Use `textfile0.txt` instead of `welcome.txt`

**Review by individual-it — CHANGES_REQUESTED** [review]
`welcome.txt` is still used
please double check if any of the tests in those files are expecting a file to exist to test "overwriting"

**Review by individual-it — APPROVED** [review]

**Pull request merged**